resources/energyplus/ProposedEnergy+.idd

```diff
@@ -42724,37 +42715,32 @@ HeatPump:WaterToWater:EIR:Heating,
    N1,  \field Load Side Reference Flow Rate
         \note This component is currently a constant-flow device, meaning it will always
         \note try to request the full design flow from the central plant manager.
-        \required-field
         \type real
         \minimum> 0.0
         \units m3/s
         \ip-units gal/min
         \autosizable
         \default autosize
    N2,  \field Source Side Reference Flow Rate
-        \required-field
         \type real
         \minimum> 0.0
         \units m3/s
         \ip-units gal/min
         \autosizable
         \default autosize
    N3,  \field Reference Capacity
-        \required-field
         \type real
         \minimum> 0.0
         \units W
         \autosizable
         \default autosize
    N4,  \field Reference Coefficient of Performance
-        \required-field
         \type real
         \minimum> 0.0
         \default 7.5
         \units W/W
```

Comment:
`IddFixture.EpIddFile` would fail because this field has a `\default` but I recently had added `\required-field` to match E+ idd...